reconcile/slack_usergroups.py

```diff
@@ -161,80 +172,50 @@ def get_slack_usernames_from_pagerduty(pagerduties, users, usergroup):
     return all_slack_usernames
 
 
-def get_slack_usernames_from_github_owners(github_owners,
-                                           github_owners_aliases, users,
-                                           usergroup):
-    return get_slack_usernames_from_owners(
-        github_owners, github_owners_aliases, users, usergroup,
-        'github_username', missing_user_log_method=logging.debug)
+def get_slack_usernames_from_owners(owners_from_repo, users, usergroup):
+    if owners_from_repo is None:
+        return []
 
+    all_slack_usernames = []
 
-def get_slack_usernames_from_gitlab_owners(gitlab_owners,
-                                           gitlab_owners_aliases, users,
-                                           usergroup):
-    return get_slack_usernames_from_owners(
-        gitlab_owners, gitlab_owners_aliases, users, usergroup,
-        'org_username', ssl_verify=False)
+    for url in owners_from_repo:
+        repo_cli = RepoCli(url)
 
+        if isinstance(repo_cli, GitLabApi):
+            user_key = 'org_username'
+            missing_user_log_method = logging.warning
+        elif isinstance(repo_cli, GithubApi):
+            user_key = 'github_username'
+            missing_user_log_method = logging.debug
```

Comment:
why the different logging severities?

Reply:
Not sure, but that's not new.

Reply:
https://github.com/app-sre/qontract-reconcile/commit/b327c9d673d5991504928dbf3692fdb918d57b26

Reply:
let's summon @maorfr then 😂

Reply:
The reason is to allow external owners in github (example: red hat employees who leave the company). In gitlab, when someone leaves, they should be removed (hence warning).